src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -243,6 +243,24 @@ void SearchAxisCompute(const nnvm::NodeAttrs& attrs,
   }
 }
 
+template<typename xpu>
+void NoGradCompute(const nnvm::NodeAttrs& attrs,
```

Comment:
@piiswrong Should I move the BlockGrad to `operator/tensor` and add also the forward pass?